### PR TITLE
Support encryption of url parameter in profiles

### PR DESCRIPTION
**Pull request by Spunkie**
Per our discussion in issue #20

### CODE REVIEW

File: lib/SpeedTracker.js [hunk 1]
```diff
@@ -315,7 +315,7 @@ SpeedTracker.prototype.runTest = function (profile, isScheduled) {
 
     if (!parameters.url) return Promise.reject('NO_URL')
 
-    let url = parameters.url
+    let url = parameters.url.trim().startsWith('http') ? parameters.url : Utils.decrypt(parameters.url, this.options.key)
```
Comment (eduardoboucas) [inline lib/SpeedTracker.js]: We're making the effort of trimming the string, which is good, but then we're not using that in the assignment if we're dealing with a plain text URL.

May I suggest something like:

```js
let url = parameters.url.trim()

if (url.startsWith('http')) {
  url = Utils.decrypt(parameters.url, this.options.key)
}
```

**Comment by eduardoboucas** [comment]
Thanks for this! 👍

**Comment by Spunkie** [comment]
Oh sorry I forgot you don't get notifications for PRs automatically with a new push. I ended up needing to rebase my repo to fix signing the commit with the wrong email. This breaks the links to our earlier line specific comments but the actual merge should be unaffected by it. I did the same and commented on my other PR as well.

**Comment by eduardoboucas** [comment]
Looks good! I'll merge this tomorrow and deploy. Thanks again!

**Comment by eduardoboucas** [comment]
Deployed to the preview instance (`dev.speedtracker.org`) and, as far as I'm concerned, ready to deploy to live if you're happy with it.

Also, do you think you could add something to the documentation at https://github.com/speedtracker/speedtracker.org?

Thanks!

**Pull request merged**